src/auth/storage.js

```diff
@@ -26,17 +26,75 @@ class MemoryStorage {
 class Storage {
   /**
    * @param {Storage} store implementer of Storage interface
-   * @param {String} namespace appID to be used for namespacing
+   * @param {String} namespace clientAppID to be used for namespacing
   */
   constructor(store, namespace) {
     this.store = store;
     this.namespace = namespace;
+    this._keySetKey = `_${this.namespace}.keys`
+  }
+  
+  _generateKey(key) {
+    return `${this.namespace}.${key}`
+  }
+  _saveKeyToKeySet(generatedKey) {
```

Comment:
[minor] Instead of storing more into local storage, why don't we just keep track of what always needs to be cleared. It's only the two keys for user auth and refresh token. I think this increases the complexity a bit too much.

Reply:
If that's the case, we can just go back to your original remark of just removing the `clear` function.

Reply:
👍